README.md

```diff
@@ -4,8 +4,8 @@
 
 Blueprint is a React UI toolkit for the Web.
 
-Palantir uses Blueprint to build complex, data-dense web interfaces for _desktop applications_.
-If you rely heavily on mobile interactions and are looking for a mobile-first UI kit, this may not be for you.
+It is optimized for building complex, data-dense Web interfaces for _desktop applications_.
+If you rely heavily on mobile interactions and are looking for a mobile-first solution, this may not be for you.
```

Comment:
If we're ok with it as a tagline, I think it's ok as a description here too. I want to avoid the term "framework" in the JS ecosystem since it suggests something more prescriptive like Angular or Ember.